charts/opentelemetry-collector/templates/daemonset.yaml

```diff
@@ -10,6 +10,18 @@ spec:
     matchLabels:
       {{- include "opentelemetry-collector.selectorLabels" . | nindent 6 }}
       {{- include "opentelemetry-collector.component" . | nindent 6 }}
+  updateStrategy:
+    {{- if eq .Values.rollout.strategy "RollingUpdate" }}
+    rollingUpdate:
```

Comment:
can we just dump `.Values.rollout.rollingUpdate` yaml here instead of all the conditions?

Reply:
👀 will see what I can do here

Reply:
Done